.github/workflows/ci_workflow.yml

```diff
@@ -0,0 +1,397 @@
+      - name: Configure
+        run: |
+          cmake ../. \
+                -DCMAKE_INSTALL_PREFIX=../_install \
+                -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} \
+                -DCMAKE_CXX_STANDARD=14 \
+                -DOCIO_BUILD_DOCS=ON \
+                -DOCIO_BUILD_GPU_TESTS=OFF \
+                -DOCIO_USE_SSE=${{ matrix.use-sse }} \
+                -DOCIO_INSTALL_EXT_PACKAGES=ALL \
+                -DOCIO_WARNING_AS_ERROR=ON \
+                -DPYTHON_EXECUTABLE=$(which python)
+        working-directory: _build
+      - name: Build
+        run: |
+          cmake --build . \
+                --target install \
+                --config ${{ matrix.build-type }} \
+                -- -j2
+        working-directory: _build
+      - name: Test
+        run: ctest -V
+        working-directory: _build
+
+  # -----------------------------------
+  # Latest ext packages (schedule only)
+  # -----------------------------------
+
+  linux_latest:
+    name: Linux CentOS 7 VFX CY2020 (latest ext packages${{ matrix.build-desc }})
+    runs-on: ubuntu-latest
+    if: github.event_name == 'schedule'
+    strategy:
+      matrix:
+        build: [1, 2]
+        include:
+          - build: 1
+            build-desc: ''
+            cxx-compiler: g++
+            cc-compiler: gcc
+          - build: 2
+            build-desc: ', Clang 7'
+            cxx-compiler: clang++
+            cc-compiler: clang
+    container:
+      image: aswf/ci-base:2020
+    env:
+      CXX: ${{ matrix.cxx-compiler }}
+      CC: ${{ matrix.cc-compiler }}
+    steps:
+      # The glew cmake modules installed in 'aswf/ci-base:2020' cause a link 
+      # failure while building OpenGLBuilder. Deleting the cmake directory 
+      # results in find_package(GLEW) finding the glew include dir instead, 
+      # resolving the failure.
+      # TODO: Investigate an aswf-docker fix and phase out this workaround
+      - name: Setup container
+        run: |
+          sudo rm -rf /usr/local/lib64/cmake/glew
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install latest ext package versions
+        run: |
+          share/ci/scripts/linux/install_expat.sh latest
+          share/ci/scripts/linux/install_lcms2.sh latest
+          share/ci/scripts/linux/install_yaml-cpp.sh latest
+          share/ci/scripts/linux/install_pystring.sh latest
+          share/ci/scripts/linux/install_pybind11.sh latest
+          share/ci/scripts/linux/install_openexr.sh latest
+          share/ci/scripts/linux/install_oiio.sh latest
+          share/ci/scripts/linux/install_sphinx.sh latest
+      - name: Create build directories
+        run: |
+          mkdir _install
+          mkdir _build
+      - name: Configure
+        run: |
+          cmake ../. \
+                -DCMAKE_INSTALL_PREFIX=../_install \
+                -DCMAKE_BUILD_TYPE=Release \
+                -DCMAKE_CXX_STANDARD=14 \
+                -DOCIO_BUILD_DOCS=ON \
```

Comment:
I do not think that the documentation generation is important here.

Reply:
.. but C++11 & C++14 are.

Reply:
Docs need to be tested here because we are also testing latest versions of pybind11 and sphinx. I have changed it to only build for one of the two jobs though.

Reply:
For now I have switched this to C++11. We can add 14 too if needed, but I was trying to keep the build count to a reasonable number. Let me know if you think 14 here is worth another two builds.